librz/core/cmd_type.c

```diff
@@ -254,6 +253,163 @@ static void types_cc_print(RzCore *core, const char *cc, RzOutputMode mode) {
 	}
 }
 
+static void types_enum_print(RzCore *core, const char *enum_name, RzOutputMode mode) {
+	rz_return_if_fail(enum_name);
+	Sdb *TDB = core->analysis->sdb_types;
+	switch (mode) {
+	case RZ_OUTPUT_MODE_JSON: {
+		RTypeEnum *member;
+		RzListIter *iter;
+		PJ *pj = pj_new();
+		RzList *list = rz_type_get_enum(TDB, enum_name);
+		pj_o(pj);
+		if (list && !rz_list_empty(list)) {
+			pj_ks(pj, "name", enum_name);
+			pj_k(pj, "values");
+			pj_o(pj);
+			rz_list_foreach (list, iter, member) {
+				pj_kn(pj, member->name, rz_num_math(NULL, member->val));
+			}
+			pj_end(pj);
+		}
+		pj_end(pj);
+		rz_cons_printf("%s\n", pj_string(pj));
+		pj_free(pj);
+		rz_list_free(list);
+	} break;
```

Comment:
put the break inside the  `{ .. }`. At least it will be with the right indentation.